### PR TITLE
Update to new TextField widget

### PR DESCRIPTION
**Pull request by collinjackson**

### CODE REVIEW

File: full/lib/main.dart [hunk 1]
```diff
@@ -145,19 +136,23 @@ class ChatScreenState extends State<ChatScreen> with TickerProviderStateMixin {
...
-          )
+          child: new TextField(
+            controller: _textController,
+            onChanged: (String text) {
+              setState(() {
+                _isComposing = text.length > 0;
+              });
+            },
+            onSubmitted: _handleSubmitted,
+            decoration: new InputDecoration(hintText: "Send a message"),
```
Comment (abarth) [inline full/lib/main.dart]: Do you want the underline?  If not, try `new InputDecoration.collapsed(...)`

**Comment by abarth** [comment]
LGTM

File: full/lib/main.dart (hunk 1 above)
Comment (collinjackson) [inline full/lib/main.dart]: good idea, thanks

**Pull request merged**